### DISCUSSION TITLE
Multiple Environments

### DISCUSSION THREAD

**Discussion by ShadowRunSucks** [almenscorner/IntuneCD#95] [General]
Wondering if there is support for multiple prod environments?

In a MSP settings and trying to manage multiple Intune tenants is it possible to push from DEV to multiple environments?
The documentation uses a standard dev-prod environment but is it possible to added additional tenants?

Thanks

**Comment by almenscorner** [comment]
Hi,

It's definitely something which is possible. The way I would go about setting that up is to have one repository that contains the baseline you want to push to the tenants. You then setup pipelines for each tenant that should use the baseline and have them checkout the baseline repo.

If you have customers who should not be affected by changes in the baseline repo you could setup a separate repository/folder for them in the same Azure DevOps organization.

**Reply by ShadowRunSucks** [reply]
Hey I finally got around to implementing a proof of concept and did exactly what you suggested and it does exactly what I want.

Unrelated question in the documentation it says it can create Update Rings but not update them but I not see them being created is there something Im missing

Thanks again

**Reply by almenscorner** [reply]
Happy to hear you were able to get it working like you envisioned :) 

Can you attach an example of the policy not being created so I can try to replicate?

**Reply by ShadowRunSucks** [reply]
[Windows Update for Business Policy _windowsUpdateForBusinessConfiguration.txt](https://github.com/almenscorner/IntuneCD/files/11338887/Windows.Update.for.Business.Policy._windowsUpdateForBusinessConfiguration.txt)


Here is the policy that is sitting in the Device Configurations folder

**Reply by almenscorner** [reply]
There definitely is a bug where it skips Update Rings all together. I will include a fix in the upcoming version so they are created when running the update!
